config/routes.rb

```diff
@@ -12,6 +12,7 @@
         get 'find_all', to: 'find#index'
         get 'find', to: 'find#show'
         get 'random', to: 'random#show'
```

Comment:
@case-eee are we doing this pluralization right? I'm getting a little lost with all of these controllers and what should be pluralized